test/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD002UseJtfRunAnalyzerTests.cs

```diff
@@ -538,6 +538,34 @@ async Task FAsync() {
         await CSVerify.VerifyCodeFixAsync(test, expected, withFix);
     }
 
+    [Fact]
+    public async Task ConfiguredTask_GetAwaiter_GetResult_ShouldReportWarning()
+    {
+        var test = @"
+using System;
+using System.Threading.Tasks;
+
+class Test {
+    void F() {
+        var task = Task.Run(() => 1);
+        task.ConfigureAwait(false).GetAwaiter().[|GetResult|]();
```

Comment:
Not familiar with this syntax? Is it specific to CSVerify to annotate the expected location of the warning?

Reply:
Yes, that's the much easier syntax to mark where the diagnostic is expected than what the prior tests use.